NAMESPACE

```diff
@@ -49,11 +51,7 @@ export(loadFromHumann)
 export(loadFromMetaphlan)
 export(loadFromMothur)
 export(loadFromQIIME2)
-export(makePhyloseqFromTreeSE)
 export(makePhyloseqFromTreeSummarizedExperiment)
-export(makeTreeSEFromBiom)
-export(makeTreeSEFromDADA2)
-export(makeTreeSEFromPhyloseq)
```

Comment:
Also these functions should still be available (although they should be in deprecate.R file)